tox.ini

```diff
@@ -29,3 +29,35 @@ commands =
     coverage report --show-missing
 deps =
     {[coverbase]deps}
+
+[testenv:securitypy2]
+basepython = python2.7
+commands =
+    bandit -r firebase_admin --skip B105
+deps =
+    bandit
+
+[testenv:securitypy3]
+basepython = python3.7
+commands =
+    bandit -r firebase_admin --skip B105
+deps =
+    bandit
+
+[testenv:lintpy3]
```

Comment:
How does flake compare with pylint? Can we remove the existing pylint stuff in favor of this?